app/javascript/controllers/projectCollaboration_controller.js

```
@@ -0,0 +1,11 @@
+import { Controller } from 'stimulus';
+import 'bootstrap-tagsinput/dist/bootstrap-tagsinput.js';
+
+export default class extends Controller {
+    embedInviteLink() {
+        setTimeout(() => {
+            $('#embedInviteLink').select();
+            document.execCommand('copy');
+        }, 100);
+    }
```

ESLint

[error] Expected 'this' to be used by class method 'embedInviteLink'.

(class-methods-use-this)